### PR TITLE
Output pane smoke test

### PR DESCRIPTION
**Pull request by testlabauto**
Basic smoke test of the output pane.  For R and Python, enters a bad line of code and then asserts that it sees a corresponding log in the output pane.

### QA Notes

All smoke tests should pass.

### CODE REVIEW

File: test/smoke/src/areas/positron/output/consoleOutputLog.test.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+export function setup(logger: Logger) {
+	describe('Output', () => {
+		// Shared before/after handling
+		installAllHandlers(logger);
+
+		describe('Console Output Log - Python', () => {
+			before(async function () {
+				const app = this.app as Application;
+				const pythonFixtures = new PositronPythonFixtures(app);
+				await pythonFixtures.startPythonInterpreter();
```
Comment (nstrayer) [inline test/smoke/src/areas/positron/output/consoleOutputLog.test.ts:19]: I wonder if this is a common enough pattern we should create a utility function for it.

Comment (nstrayer) [reply test/smoke/src/areas/positron/output/consoleOutputLog.test.ts:19]: E.g. 
```ts

export class PositronRFixtures {

	static async SetupFixtures(app: Application) {
		const fixtures = new PositronRFixtures(app);
		await fixtures.startRInterpreter();
	}
	...
}
```

And then 
```ts
...
before(async function () {
    await PositronRFixtures.SetupFixtures(this.app as Application);
});
...
```
Probably premature optimization though.

**Review by nstrayer — DISMISSED** [review]
This looks great to me!

Added some comments just in general but more out of curiosity than anything else.

File: test/automation/src/positron/positronOutput.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+		await this.quickinput.waitForQuickInputOpened();
+		await this.quickinput.type(outputPaneNameContains);
+
+		await this.quickinput.selectQuickInputElementContaining(outputPaneNameContains);
+		await this.quickinput.waitForQuickInputClosed();
+	}
+
+	async waitForOutContaining(fragment: string) {
+		await this.code.waitForElements(OUTPUT_LINE, false, (contents) => contents.some(line => line.textContent.includes(fragment)));
```
[the remark this answers is not in the document]
Comment (testlabauto) [reply test/automation/src/positron/positronOutput.ts:31]: Cool idea.  It does work almost the same but the isVisible implementation does have a problem with the fact that two logs both match the fragment whereas the evaluate implementation seems to be fine with ignoring the extra.  Not sure which is preferable, actually.

File: test/smoke/src/areas/positron/output/consoleOutputLog.test.ts (hunk 1 above)
Comment (testlabauto) [reply test/smoke/src/areas/positron/output/consoleOutputLog.test.ts:19]: Yeah, those fixtures are used in most tests.  i will change to use the static approach.

File: test/automation/src/positron/positronOutput.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (testlabauto) [reply test/automation/src/positron/positronOutput.ts:31]: I will just do it like this:
```
async waitForOutContaining(fragment: string) {
		const outputLine = this.code.driver.getLocator(OUTPUT_LINE);
		await outputLine.getByText(fragment).first().isVisible();
	}
```
That way we can hold onto Nick's approach here for future reference as its an alternate approach to waitForElements.

File: test/automation/src/positron/positronOutput.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+	async openOutputPane(outputPaneNameContains: string) {
+		await this.quickaccess.runCommand('workbench.action.showOutputChannels', { keepOpen: true });
+
+		await this.quickinput.waitForQuickInputOpened();
+		await this.quickinput.type(outputPaneNameContains);
+
+		await this.quickinput.selectQuickInputElementContaining(outputPaneNameContains);
+		await this.quickinput.waitForQuickInputClosed();
+	}
```
[the remark this answers is not in the document]
Comment (testlabauto) [reply test/automation/src/positron/positronOutput.ts:28]: Thanks!

**Review by nstrayer — APPROVED** [review]
Looks great to me!

**Review by jonvanausdeln — APPROVED** [review]
LGTM! Works on my windows box!

**Pull request merged**